tooling/cli/src/command/tokens/config.ts

```diff
@@ -7,6 +7,7 @@ export const themeKeyConfiguration: ThemeKeyOptions[] = [
   { key: "fonts" },
   { key: "fontSizes" },
   { key: "fontWeights" },
+  { key: "layerStyles" },
```

Comment:
This config file is only used by `extractPropertyPath`. Leaving `layerStyles` and `textStyles` in there could confuse other contributors. (Seems like I left `textStyles` in there, but it does nothing)

Would you kindly remove both entries?

Reply:
Ah yeah of course, done!